Tools/TestWebKitAPI/Tests/WebKit/AdvancedPrivacyProtections.mm

```diff
@@ -849,7 +858,7 @@ HTTPServer server({
     EXPECT_FALSE([values.first isEqualToString:values.second]);
 }
 
-TEST(AdvancedPrivacyProtections, VerifyPixelsFromNoisyCanvas2DAPI)
+TEST_F(AdvancedPrivacyProtections, VerifyPixelsFromNoisyCanvas2DAPI)
```

Comment:
I don't think these fingerprinting tests rely on WebPrivacy.framework. Can we keep these as `TEST(…)`?